cmd/apprepository-controller/controller.go

```diff
@@ -553,38 +558,38 @@ func cronJobName(apprepo *apprepov1alpha1.AppRepository) string {
 }
 
 // deleteJobName returns a unique name for the Job to cleanup AppRepository
-func deleteJobName(reponame, reponamespace string) string {
-	return fmt.Sprintf("apprepo-%s-cleanup-%s", reponamespace, reponame)
+func deleteJobName(reponame string, config Config) string {
+	return fmt.Sprintf("apprepo-%s-cleanup-%s", config.KubeappsNamespace, reponame)
 }
```

Comment:
be careful with this, `reponamespace` is the namespace in which the app repository is created, here (and in other places), it's expected to use the repository namespace, not the namespace in which kubeapps is installed.

Reply:
Thanks for pointing this out! I've changed it a little bit for avoiding further confusion: now the `apprepo` is directly being passed through instead of manually passing the repoName and repoNamespace strings